packages/components/pie-link/src/index.ts

```diff
@@ -50,7 +56,9 @@ export class PieLink extends LitElement implements LinkProps {
                 rel=${ifDefined(rel)}
                 ?isBold=${isBold}
                 ?isStandalone=${isStandalone}>
+                <slot name="icon-leading"></slot>
                 <slot></slot>
+                <slot name="icon-trailing"></slot>
```

Comment:
Just to check – will we add validation if people try to add both a leading and a trailing icon as a separate PR?

Reply:
I'm not against leaving at is so there's a note or guidence now. Also we would have docs for this at some point too so we can reiterate it there too.

Reply:
Instead of validation what if we just add a single icon slot and use a prop to determine if it is leading or trailing? I personally don't think we should allow both icons to be present at the same time if this goes against the design system rules

Reply:
sorry I don't think I was in when this ticket was refined but I quite like Jamie's idea 😄 I'd probably lean towards only allowing one, just to avoid any design inconsistencies in case people miss the docs and start adding both icons

Reply:
I don't think we should rely on users doing the right thing for this (as much as I want to believe they'll read our docs 😆 ) – I think we need to restrict it in some way.